### ISSUE TITLE
[21.4.1] release version reports itself as 21.4.1~dev1

### ISSUE THREAD

**Issue by tgurr** [greenbone/gsa#3048]
### Expected behavior

```
# gsad --version
Greenbone Security Assistant 21.4.1
```

### Actual behavior / Steps to reproduce

```
# gsad --version
Greenbone Security Assistant 21.4.1~dev1
```

The version displayed on the GSA webinterface `/about` page however is correctly displayed as `Version (21.4.1)`.

### GVM versions

**gsa:** Greenbone Security Assistant 21.4.1~dev1
**gvm:** Greenbone Vulnerability Manager 21.4.2 / Manager DB revision 242
**openvas-scanner:** OpenVAS 21.4.1
**gvm-libs:** gvm-libs 21.4.1

### Environment

**Operating system:** `Linux exherbo 5.12.0 #1 SMP Thu Apr 29 14:22:53 CEST 2021 x86_64 GNU/Linux`
**Installation method / source:** Exherbo packages (source-based)

### Logfiles
gsad.log: `gsad main:MESSAGE:2021-06-30 15h26.14 utc:337: Starting GSAD version 21.4.1~dev1`

**Comment by carismalife** [comment]
hi
Is anyone solve the problem?

**Comment by bjoernricks** [comment]
This will be fixed with the next release planned for the last week of July.

**Comment by carismalife** [comment]
Thank you

iPhone’umdan gönderildi

**Comment by carismalife** [comment]
Hello Björn,

Are there any workaround solutions to make scan before last week of july? 

Thank you

Murat

iPhone’umdan gönderildi

**Comment by bjoernricks** [comment]
Yes `PROJECT_DEV_VERSION` needs to be set to false https://github.com/greenbone/gsa/blob/gsa-21.04/CMakeLists.txt#L35

**Comment by cfi-gb** [comment]
>  Are there any workaround solutions to make scan before last week of july?

Please do note that this is just a cosmetic thing and doesn't affect the functionality of GVM like e.g. scans at all.

**Comment by carismalife** [comment]
Thank you very much Björn 

iPhone’umdan gönderildi

**Comment by carismalife** [comment]
Hi again Björn

So my problem is different as i got it from you previos post.

When i make scan my settings are below but nothing find any vuln.

Port list:All IANA assigned tcp
Allive test: Scan config default
Scanner:openVas Default
Scan Config: Full and fast

When the scan finished ex: below Log column are ok but other high,medium and low are “0”. I know one of the system  %100 sure vulnerable but not find it. Please help me, i will be appreciate.

Thank you

iPhone’umdan gönderildi

**Comment by cfi-gb** [comment]
@carismalife Please use https://community.greenbone.net for configuration / installation / support questions.

**Comment by bjoernricks** [comment]
This should be fixed with the 21.4.2 release.
